File: Leetcode/9. Sorting/Bubble Sort.py
def bubbleSort(nums):
    for i in range(len(nums)):
        for j in range(len(nums)-i-1):
            if nums[j] > nums[j+1]:
                nums[j], nums[j+1] = nums[j+1], nums[j]

    return nums


print(bubbleSort([9,8,7,6,5,4,3,2,1]))

File: Leetcode/9. Sorting/Bubble Sort.js
function bubbleSort(nums) {
  for (let i = 0; i < nums.length; i++) {
    for (let j = i + 1; j < nums.length; j++) {
      if (nums[i] > nums[j])
        return (nums[j], (nums[j + 1] = nums[j + 1]), nums[j]);
    }
  }
}

console.log(bubbleSort([9, 8, 7, 6, 5, 4, 3, 2, 1]));
